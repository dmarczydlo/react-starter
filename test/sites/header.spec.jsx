import React from 'react';
import Header from '../../src/layout/header';

describe('Header', () => {
    let wrapper;

    it('should be rendered', () => {
        wrapper = shallow(
            <Header/>
        );
        expect(wrapper).toMatchSnapshot();
    });
});