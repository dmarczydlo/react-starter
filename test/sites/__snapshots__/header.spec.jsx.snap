// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should be rendered 1`] = `
<Fragment>
  <div>
    Header
  </div>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <meta
      charSet="utf-8"
    />
    <title>
      React starter
    </title>
  </HelmetWrapper>
</Fragment>
`;
